catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -153,9 +153,8 @@ public String addRegistryEntry(Metacard metacard) throws FederationAdminExceptio
             }
             registryIds = createResponse.getCreatedMetacards()
                     .stream()
-                    .map(e -> e.getAttribute(RegistryObjectMetacardType.REGISTRY_ID)
-                            .getValue()
-                            .toString())
+                    .filter(RegistryUtility::isRegistryMetacard)
+                    .map(RegistryUtility::getRegistryId)
```

Comment:
Ignore. On second look I see that `isRegistryMetacard` will already account for that.